cli/enrol.yaml

```diff
@@ -0,0 +1,72 @@
+## Every recipe file must provide at least the component property.
+## Based on the plugin type plugin specific files files will be generated.
+component: enrol_plugintest
+
+## Human readable name of the plugin.
+name: Enrol plugin
+
+## Human readable release number.
+release: "0.1.0"
+
+## Plugin version number, e.g. 2016062100. Will be set to current date if left empty.
+#version: 2016121200
+
+## Required Moodle version, e.g. 2015051100 or "2.9".
+requires: "2.9"
+
+## Plugin maturity level. Possible options are MATURIY_ALPHA, MATURITY_BETA,
+## MATURITY_RC or MATURIY_STABLE.
+maturity: MATURITY_BETA
+
+## Copyright holder(s) of the generated files and classes.
+copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>
+
+## Features flags can control generation of optional files/code fragments.
+features:
+    readme: true
+    license: true
+    install: true
+    settings: true
+
+## Allows manual enrolment. Requires enrol/<enrolname>:enrol capability.
+allow_enrol: 'false'
```

Comment:
How about renaming '<plugintype>_features' to simply <plugintype>?

The recipe would look like this:
```
## Required Moodle version, e.g. 2015051100 or "2.9".
requires: "2.9"

## Plugin maturity level. Possible options are MATURIY_ALPHA, MATURITY_BETA,
## MATURITY_RC or MATURIY_STABLE.
maturity: MATURITY_BETA

## Copyright holder(s) of the generated files and classes.
copyright: 2016 Alexandru Elisei <alexandru.elisei@gmail.com>, David Mudrák <david@moodle.com>

## Features flags can control generation of optional files/code fragments.
features:
  readme: true
  license: true

  ## Creates the file settings.php. The function has_config() in
  ## block_<blockname>.php will return true.
  #settings: true

qtype:
    base_class: question_graded_automatically
    feature1: true
    feature2: false

strings:
    - id: pluginnamesummary
      text: Rendered when choosing the question type to be added.
    - id: pluginnameadding
      text: Rendered when adding this question type.
    - id: pluginnameediting
      text: Rendered when editing this question type.
    - id: pluginname_help
      text: Help text when adding this question type.
```